javascript/app.js

```diff
@@ -1,14 +1,16 @@
-const menuBars = document.getElementById('menu-bars');
-const mobileMenu = document.getElementById('mobile-menu');
-const closeMenuBtn = document.getElementById('close-menu-btn');
-const mobileLinks = document.getElementById('mobile-links');
-function openMenuHandler() {
-  mobileMenu.classList.replace('d-none', 'd-flex');
-}
-function closeMenuHandler() {
-  mobileMenu.classList.replace('d-flex', 'd-none');
-}
-
-menuBars.addEventListener('click', openMenuHandler);
-closeMenuBtn.addEventListener('click', closeMenuHandler);
-mobileLinks.addEventListener('click', closeMenuHandler);
+const menuBars = document.getElementById('menu-bars');
+const mobileMenu = document.getElementById('mobile-menu');
+const closeMenuBtn = document.getElementById('close-menu-btn');
+const mobileLinks = document.getElementById('mobile-links');
+
+function openMenuHandler() {
+  mobileMenu.classList.replace('d-none', 'd-flex');
+}
+
+function closeMenuHandler() {
+  mobileMenu.classList.replace('d-flex', 'd-none');
+}
+
+menuBars.addEventListener('click', openMenuHandler);
+closeMenuBtn.addEventListener('click', closeMenuHandler);
+mobileLinks.addEventListener('click', closeMenuHandler);
```

Comment:
Great job implementing the figma design using the project requirements, please kindly make the mobile menu display background colour to be the same with the Figma design. While you really tried with your design, your background colour is darker and it covers the text compared to what we have in the design, kindly implement this change as it will ensure legible text interface

Your design :
![image](https://github.com/hafedEfheij/PortfolioPoject/assets/20537712/154d14b2-0df6-4083-844a-776d598bac9c)


Figma design :
![image](https://github.com/hafedEfheij/PortfolioPoject/assets/20537712/667ead64-b988-4631-9bb3-98144ce9b14d)